content/en/blog/_posts/2022-07-27-crd-validation-rules-graduate-to-beta.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: blog
+title: "Kubernetes 1.25: CustomResourceDefinition Validation Rules Graduate to Beta"
+date: 2022-07-27
+slug: tbd
+canonicalUrl: tbd
+---
+
+**Authors:** Joe Betz (Google), Kermit Alexander (Google)
+
+In Kubernetes 1.25, [Validation rules for CustomResourceDefinitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) (CRDs) have graduated to Beta!
```

Comment:
Ideally, link to the section about validation rules in the CRD concept docs (if there isn't a section yet, please mention the PR that'll add it so that a reviewer can cross-check).

Reply:
Good catch. We don't have the section yet. I'll get that sorted out and fix this link.

Reply:
This might be too much to fix in time for this blog entry, but I've opened https://github.com/kubernetes/website/issues/35991 to address the larger issue here.